docs/install.rst

```diff
@@ -90,6 +93,11 @@ Disable and remove the swapfile once the virtual enviroment is constructed::
   user@webserver:~$ sudo swapoff /tmp/swapfile
   user@webserver:~$ sudo rm /tmp/swapfile
 
+Custom Python environment
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Note that if you have configured any custom Python environment variables on your system, make sure they are properly (un)set when running `certbot-auto` and `sudo`. This could happen if you run `sudo zsh` as your user's `PYTHONPATH` will be preserved which might cause problems.
```

Comment:
@bmw Your changes are fine.  However, you can set up virtual environments to link to all the system wide libraries as well. It is not the default, but it is possible.